ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json

```diff
@@ -1587,6 +1587,34 @@
       }
     ]
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createCaseLink",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-publiclaw-courtadmin",
```

Comment:
Looking at steps 4 h and i do we not need to grant hmcts-staff  and hmcts-judiciary access to these (through the RolesToAccessProfiles). Think these roles get added in work allocation (so not live quite yet).

Reply:
I think this bit will be done as part of https://tools.hmcts.net/jira/browse/DFPL-1688 so I implemented a bit of a temporary solution for now so it can be tested

Reply:
ok.

is it definitely just court admin and superuser we want to be able to do this? e.g. would a judge want to be able to link a case?

more generally think it would be worth doing a demo to service team to get some feedback from them.